ivy/functional/backends/numpy/experimental/losses.py

```diff
@@ -75,29 +75,6 @@ def soft_margin_loss(
         return loss
 
 
-@with_unsupported_dtypes({"1.26.0 and below": ("bool", "bfloat16")}, backend_version)
```

Comment:
Same here. You can't remove backend implementation from any backend unless you are creating a new implementation that is better